depends/packages/systemtap.mk

```
@@ -0,0 +1,15 @@
+package=systemtap
+$(package)_version=4.5
+$(package)_download_path=https://sourceware.org/systemtap/ftp/releases/
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+$(package)_sha256_hash=75078ed37e0dd2a769c9d1f9394170b2d9f4d7daa425f43ca80c13bad6cfc925
+$(package)_patches=remove_SDT_ASM_SECTION_AUTOGROUP_SUPPORT_check.patch
+
+# Only the includes/sys/sdt.h header is needed from Systemtap.
+# The configure and make steps and not required.
+
+define $(package)_preprocess_cmds
+  patch -p1 < $($(package)_patch_dir)/remove_SDT_ASM_SECTION_AUTOGROUP_SUPPORT_check.patch && \
+  mkdir -p $($(package)_staging_prefix_dir)/include/sys && \
+  cp -r includes/sys $($(package)_staging_prefix_dir)/include
```

Given we aren't using sdt-config.h, there's no reason to copy sdt-config.h.in into our includes dir. This could be simplified to:

```suggestion
  cp includes/sys/sdt.h $($(package)_staging_prefix_dir)/include/sys/sdt.h
```

This would also make it clear that the only thing we care about is the sdt header.